### DISCUSSION TITLE
How to stake KTON using KtonStakingV2 and participate in KtonDAO?

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1565] [General]
1. Vist KtonStakingV1 and connect to you wallet on Crab chain:

https://kton-staking.darwinia.network/

claim, and then unstake your CKTON.

2. Vist KtonStakingV2 and connect to you wallet on Crab chain

https://kton-staking-v2.darwinia.network/

Approve and stake, you will get a non-transferrable gCKTON for voting on https://www.tally.xyz/gov/cktondao

3. Wait for a while(Tally need to finish indexing before showing your gCKTON balance), visit https://www.tally.xyz/gov/cktondao and connect to you wallet on Crab chain

Delegate to your self or someone else before this delegate do voting.

**Comment by SasoLithops** [comment]
Great to see this. I am working on proposing idea for KTON DAO. Is this implemented to Kton as well? I was able to connect to darwinia as well. Looking forward for both DAOs to start working.

**Reply by hackfisher** [reply]
Not for Darwinia yet, Darwinia's Runtime 6650 is not upgraded yet, even though you can see it, it will not work(no reward to claim).

**Reply by SasoLithops** [reply]
Understood 👍

**Comment by hackfisher** [comment]
https://www.tally.xyz/gov/ktondao KtonDAO is online.
